src/v/test_utils/tmp_dir.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "seastarx.h"
+#include "vlog.h"
+
+#include <seastar/util/log.hh>
+#include <seastar/util/tmp_file.hh>
+
+#include <exception>
+
+namespace details {
+static inline ss::logger tmpdir_logger("tmpdir-log");
```

Comment:
I think you want to drop `static`. Presumably if this were included in multiple `.cc` files the same log name would be registered twice? If it is `inline` then past something like c++17 will guarantee a single instance of the symbol.

Reply:
fixed that